### PR TITLE
Add ConstansCore

### PR DESCRIPTION
**Pull request by samuelloza**

### CODE REVIEW

File: src/main/java/org/fundacionjala/core/constans/ConstantsCore.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package org.fundacionjala.core.constans;
+
+/**
+ * [SL] This class offers constant for the core.
+ */
+public final class ConstantsCore {
```
Comment (MirkoFer-622) [inline src/main/java/org/fundacionjala/core/constans/ConstantsCore.java]: What if this class should be an enum instead?

File: src/main/java/org/fundacionjala/core/constans/ConstantsCore.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    public static final String BROWSER_CHROMIUM = "Chromium";
+    public static final String BROWSER_MOZILLA = "Mozilla Firefox";
+    public static final String BROWSER_EDGE = "Microsoft Edge";
+
+    /** Browser Properties Constants. **/
+    public static final String BROWSER_VERSION = "version";
+    public static final String BROWSER_IMPLICITWAIT = "implicitWaiting";
+    public static final String BROWSER_EXPLICITWAIT = "explicitWaiting";
+    public static final String BROWSER_SLEEPINGTIME = "sleepingTime";
+    public static final String BROWSER_FLAGS = "flags";
```
Comment (MirkoFer-622) [inline src/main/java/org/fundacionjala/core/constans/ConstantsCore.java]: Why are they necesary to save this constants? they are saved as Browser's class attributes, so to access them we have the getters. Please delete these constants.

Comment (samuelloza) [reply src/main/java/org/fundacionjala/core/constans/ConstantsCore.java]: That is correct,  I delete these constants

File: src/main/java/org/fundacionjala/core/constans/ConstantsCore.java (hunk 1 above)
Comment (samuelloza) [reply src/main/java/org/fundacionjala/core/constans/ConstantsCore.java]: I think the best solution is change name maybe to BrowserConstans.

**Review by MirkoFer-622 — APPROVED** [review]
LGTM!

**Pull request merged**